### ISSUE TITLE
Formatting issue in border

### ISSUE THREAD

**Issue by bhaap** [andrey-git/home-assistant-custom-ui#82]
### Checklist
- [*] I'm running the latest version of CustomUI ([Update guide](https://github.com/andrey-git/home-assistant-custom-ui/blob/master/docs/installing.md#updating)) or using a specific [release](https://github.com/andrey-git/home-assistant-custom-ui/releases) that is not marked as "Broken".
- [*] I tried to force-refresh (Ctrl+Shift+R / Ctrl+F5) the browser
- [*] (Optional, but recommended) I'm using Chrome or tried to reproduce the feature on Chrome.

**Browser + Version:** Chrome 63.0.3239.132 

**CustomUI version:** 20180112 

**Home Assistant release (`hass --version`):** 0.59.2 

**Problem-relevant `configuration.yaml` entries:**
```yaml
  customize_glob:
    "*.*":
      custom_ui_state_card: state-card-custom-ui  
    sensor.temperatur_koket:
      state_card_mode: badges
    sensor.ljus_koket:
      state_card_mode: badges
    sensor.jordfuktighet_koket:
      state_card_mode: badges
    sensor.vaxtnaring_koket:
      state_card_mode: badges
```

**Any errors from browser Javascript console:**
```
frontend-a11afc34fee98099d1a916297a146c46.html:57 paper-drawer-panel is deprecated. Please use app-layout instead!
ready @ frontend-a11afc34fee98099d1a916297a146c46.html:57
[Deprecation] Styling master document from stylesheets defined in HTML Imports is deprecated, and is planned to be removed in M65, around March 2018. Please refer to https://goo.gl/EGXzpw for possible migration paths.
state-card-custom-ui-dbg-es5.html:364 Loaded CustomUI 20180112
group.nedervaning_view:1 Uncaught (in promise) DOMException: Only secure origins are allowed (see: https://goo.gl/Y0ZkNV).
```

<img width="299" alt="skarmavbild 2018-01-13 kl 01 03 03" src="https://user-images.githubusercontent.com/2168003/34900910-6580d7c0-f804-11e7-820d-21fb1d2d30a0.png">

**Comment by andrey-git** [comment]
I don't understand. What the problem?

**Comment by bhaap** [comment]
![img_4324](https://user-images.githubusercontent.com/2168003/35155728-e5ad66dc-fd2e-11e7-8633-137bdbbe6c79.jpg)
You see that the border gets weird colors.

I think is not noticeable when using the default theme because border and window is the same color.

**Comment by andrey-git** [comment]
The inner color is `--paper-card-background-color`
The outer color is `--secondary-background-color`
I can't reproduce a different color for a border

**Comment by bhaap** [comment]
maby --paper-listbox-background-color

**Comment by bhaap** [comment]
this is my theme:
```
    PmxMononight:
  # MyVariables
      base-hue: '40' #Controls the base (and accent) color hue (0-360) | 0=Red 60=Yellow 120=Green 180=Cyan 240=Blue 300=Magenta 360=Red
      base-sat: '16%' #Controls the saturation of the theme (0%-100%) | 0%=Grey 100%=Full Saturation
  # MyVar
      huesat: 'var(--base-hue), var(--base-sat),'
  # Primary Color
      primary-color: 'hsl(var(--base-hue), 90%, 50%)'
  # Backgrounds
      primary-background-color: 'hsl(var(--huesat) 20%)'
      secondary-background-color: 'hsl(var(--huesat) 16%)'
      paper-listbox-background-color: 'hsl(var(--huesat) 20%)'
      paper-card-background-color: 'hsl(var(--huesat) 12%)'
      paper-dialog-background-color: 'var(--paper-card-background-color)'
      table-row-background-color: 'hsl(var(--huesat) 12%)'
      table-row-alternative-background-color: 'hsl(var(--huesat) 10%)'
  # Devider
      divider-color: 'hsla(0, 0%, 0%, 0)'
      dark-divider-opacity: '0'
      light-divider-opacity: '0'
  # Text colors
      primary-text-color: 'hsl(var(--huesat) 60%)'
      text-primary-color: 'hsl(var(--huesat) 60%)'
      secondary-text-color: 'hsl(var(--huesat) 60%)'
      disabled-text-color: 'hsl(var(--huesat) 70%)'
      sidebar-text_-_color: 'hsl(var(--huesat) 90%)'
      sidebar-text-color: 'hsl(var(--huesat) 90%)'
      paper-card-header-color: 'hsl(var(--base-hue), 90%, 50%)'
  # Text Adjustments
      paper-font-headline_-_letter-spacing: '-0.5px'
      paper-font-headline_-_font-weight: '500'
      paper-font-body1_-_font-weight: '500'
  # Nav Menu
      paper-listbox-color: 'hsl(var(--huesat) 50%)'
      paper-grey-50: 'hsl(var(--huesat) 50%)'
      paper-grey-200: 'hsla(var(--huesat) 26%)'
  # Paper card
      paper-item-icon-color: 'hsl(var(--huesat) 30%)'
      paper-item-icon-active-color: 'var(--paper-item-icon-color)'
      paper-item-icon_-_color: 'var(--paper-item-icon-color)'
      paper-item-selected_-_background-color: 'hsla(0, 0%, 0%, 0.2)'
      paper-tabs-selection-bar-color: 'hsla(0, 0%, 0%, 0.2)'
  # Labels
      label-badge-red: 'hsla(0, 0%, 0%, 0)'
      label-badge-border-color: 'var(--label-badge-red)'
      label-badge-background-color: 'var(--paper-card-background-color)'
      label-badge-text-color: 'var(--primary-text-color)'
  # Shadows
      shadow-elevation-2dp_-_box-shadow: 'inset 0px 0px 0px 4px hsl(var(--huesat) 18%)'
      shadow-elevation-16dp_-_box-shadow: 'inset 0px 0px 0px 4px hsl(var(--huesat) 28%)'
  # Switches
      paper-toggle-button-checked-button-color: 'hsl(var(--base-hue), 90%, 50%)'
      paper-toggle-button-checked-ink-color: 'hsl(var(--base-hue), 90%, 50%)'
      paper-toggle-button-checked-bar-color: 'hsl(var(--huesat) 25%)'
      paper-toggle-button-unchecked-button-color: 'hsl(var(--huesat) 25%)'
      paper-toggle-button-unchecked-ink-color: 'hsl(var(--huesat) 25%)'
      paper-toggle-button-unchecked-bar-color: 'hsl(var(--huesat) 5%)'
  # Sliders
      paper-slider-knob-color:  'hsl(var(--base-hue), 90%, 50%)'
      paper-slider-knob-start-color: 'hsl(var(--base-hue), 80%, 25%)'
      paper-slider-pin-color:  'hsl(var(--base-hue), 90%, 50%)'
      paper-slider-active-color:  'hsl(var(--base-hue), 90%, 50%)'
      paper-slider-container-color: 'hsl(var(--huesat) 28%)'
      paper-slider-secondary-color: 'hsl(var(--huesat) 90%)'
      paper-slider-disabled-active-color: 'hsl(var(--base-hue), 80%, 25%)'
      paper-slider-disabled-secondary-color: 'hsl(var(--base-hue), 80%, 25%)'
      paper-dialog-color: 'hsl(var(--base-hue), 20%, 80%)'
```

**Comment by andrey-git** [comment]
This is `shadow-elevation-2dp_-_box-shadow: 'inset 0px 0px 0px 4px hsl(var(--huesat) 18%)'`
The "inset" part is drawing a shadow inside the box.

**Comment by bhaap** [comment]
Ah ok. And this shadow somehow is under some custom_ui box?

**Comment by andrey-git** [comment]
This issue is unrelated to customui.
ha-card has `box-shadow: var(--shadow-elevation-2dp_-_box-shadow);`

**Comment by bhaap** [comment]
Well the interuption of the shadow came with customui. So there must be somthing with customui making the border this way.

**Comment by andrey-git** [comment]
I can't reproduce it.

Also is the screenshot of a card or is it the more-info window.
Could you post the config of the relevant card?

**Comment by bhaap** [comment]
Thats a card not a more-info window, the issue began when i made the glob customization below. 
If I comment out the entire customize_glob: the issue is gone, and also the use of custom ui.

The info of the card right:

```
  kok:
    name: Kök
    entities:
      - sensor.temperatur_koket
      - sensor.ljus_koket
      - sensor.jordfuktighet_koket
      - sensor.vaxtnaring_koket
      - binary_sensor.norrfonster_koket
      - binary_sensor.frysen
      - sensor.frystemp_temperature
      - binary_sensor.kylen
      - sensor.kyltemp_temperature
      - binary_sensor.diskmaskin
      - sensor.diskmaskin_energy
      - switch.taklampa_kok
      - light.kokbordlampa
      - light.kokbordlamprgb
      - switch.eluttag3
```
```
  customize_glob:
    "*.*":
      custom_ui_state_card: state-card-custom-ui
    sensor.temperatur_koket:
      state_card_mode: badges
    sensor.ljus_koket:
      state_card_mode: badges
    sensor.jordfuktighet_koket:
      state_card_mode: badges
    sensor.vaxtnaring_koket:
      state_card_mode: badges
    light.*:
      state_card_mode: break-slider
      stretch_slider: true
      slider_theme:
        pin: true
```

Is there any more info you need.

**Comment by bhaap** [comment]
Issue resolved after update of home assistant to 0.63 and customui to 20180216
